cmd/common.go

```diff
@@ -61,13 +66,18 @@ func assertInFlutterProject() string {
 			goto Fail
 		}
 
-		return pubspec.Name
+		if pubspec.Author == "" {
+			fmt.Println("hover: Missing `author` in pubspec.yaml.")
+			goto Fail
```

Comment:
Perhaps this is required for packaging, but it is not for a simple build / run. This should not be in assertInFlutterProject().